dm/syncer/data_validator.go

```diff
@@ -198,3 +399,158 @@ func (v *DataValidator) Stage() pb.Stage {
 	defer v.RUnlock()
 	return v.stage
 }
+
+func (v *DataValidator) startValidateWorkers() {
+}
+
+func (v *DataValidator) dispatchRowChange(key string, row *rowChange) {
+}
+
+func (v *DataValidator) processRowsEvent(header *replication.EventHeader, ev *replication.RowsEvent) error {
+	sourceTable := &filter.Table{
+		Schema: string(ev.Table.Schema),
+		Name:   string(ev.Table.Table),
+	}
+	fullTableName := sourceTable.String()
+	if _, ok := v.unsupportedTable[fullTableName]; ok {
+		return nil
+	}
+
+	targetTable := v.syncer.route(sourceTable)
+	tableInfo, err := v.syncer.schemaTracker.GetTableInfo(sourceTable)
```

Comment:
here we get TableInfo of upstream, not the DownstreamTableInfo, so we need to handle PK IndexInfo is not in Indices when PkIsHandle

Reply:
see https://github.com/pingcap/tiflow/pull/4539#discussion_r815644442, will left a todo

Reply:
I'm OK with left a TODO.

Another problem is I'm not sure `GetTableInfo` will race when the TableInfo is changed because syncer is processing a DDL.